ports/libgwenhywfar/portfile.cmake

```
@@ -0,0 +1,65 @@
+set(VERSION_MAJOR 5)

```

```suggestion
vcpkg_fail_port_install(ON_TAREGT "osx" "linux")

set(VERSION_MAJOR 5)
```
It would be better to add the unsupported message here.

---

It's not supported on Windows, not osx or linux. And there's already a `"supports": "!windows",` in `vcpkg.json`. Is it still beneficial adding additional check to the `portfile.cmake`?